contracts/token/erc1155/README.md

```diff
@@ -0,0 +1,21 @@
+# ERC1155 contracts
+
+The ImmutableERC1155 contracts allows clients to mint multiple different tokens with different token ids within the same collection. The contract features methods to allow for minting multiples of multiple token ids to simplify the minting flow and reduce gas costs. 
+
+[Read more On the Threat Model](../../../audits/202312-threat-model-preset-erc1155.md)
+
```

Comment:
What about a link to the standard here for more info: https://eips.ethereum.org/EIPS/eip-1155